Automattic/wp-calypso
# Launchpad :: add newsletter steps (stripe/paid subscriptions)

Related to #72651, P2: p81Rsd-1dU-p2

## Proposed Changes

* Add stripe account connected step to newsletter flow.
* Add paid newsletter plan to newsletter flow.

## Screenshot
See "Connect Stripe account" and "Create a paid newsletter".

<img src="https://user-images.githubusercontent.com/6354169/229555024-30562def-4267-4e1e-ae3c-836e7f4bc503.png" width=300 />

## Testing Instructions



* Checkout this branch
* Sandbox public-api.wordpress.com & store: PCYsg-IA-p2.
* Start a new newsletter site from - http://calypso.localhost:3000/setup/newsletter/intro
* Checkout
* Verify that the steps now include connecting a stripe account and creating a paid plan.
* Following those steps should mark off the corresponding options.
* Returning from the stripe flow kicks you onto wordpress.com.  You will have to mangle the URL to return to your localhost environment -- which won't happen once this PR is merged. (i.e. `http://calypso.localhost:3000/earn/payments/${siteSlug}?stripe_connect_success=launchpad`).  Verify that the url param is correct.

## Pre-merge Checklist
